analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Operations.cs

```diff
@@ -254,6 +254,48 @@ public void Conversion_CustomOperators_DoNotPropagateState_Half()
 
 #endif
 
+        [TestMethod]
+        public void Conversion_ToLocalNonNullableValueType_CS()
+        {
+            var validator = SETestContext.CreateCS("""
+                object o = null; // Set ObjectConstraint.Null and Dummy (LiteralDummyTestCheck)
+
+                int convert = System.Convert.ToInt32(o);
```

Comment:
This is the equivalent for VBs `CType`. At least converter.telerik.com thinks so. Sharplab shows in the decompiled source that [Microsoft.VisualBasic.CompilerServices.Conversions.ToInteger](https://learn.microsoft.com/dotnet/api/microsoft.visualbasic.compilerservices.conversions.tointeger?view=net-7.0#microsoft-visualbasic-compilerservices-conversions-tointeger(system-object)) is used: [sharplab](https://sharplab.io/#v2:EYLgtghglgdgNAGxAN2HAJiA1AHwJJgAOA9gE4AuAzgAQDKAnpeQKZgCwAUAAoCuwCUAMbUAssXQ8Ezal1LEA5qQjsO1NTL4DhtPqOgwAFAEpO6s9QAiUMNWLUAgjQDywAFbNB5agF5qAOWJyAAtYeWoAcjpmLxd3TwBhYhgmJVhyADo/SQRqCBh0Sx4wMHpqAwAZKBYlBAsikoAVZiZ4oI8AaxNVczUrGyh4hvpCaUdqPBgWeWZSUx7qAaGRn2pB4eYDYjhxyeZp0i6zAFF8uj5OE4KxCSkgA==). But for the purpose shown here it doesn't make a difference: The conversion is done via helper methods and the constraints are not propagated.